### ISSUE TITLE
Using left arrow to lower a Rating does not work when selected by tabbing

### ISSUE THREAD

**Issue by PaulWells** [winjs/winjs#1365]
Repro: In Edge or IE open the try site and in the first rating example click on the first rating.  Tab down to the fifth Rating then press the right arrow key to increase the rating.  When you press the left arrow key, the rating will not decrease.  This repros in the dev branch as well.
